src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -635,12 +635,12 @@ static void xclmgmt_clock_get_data_impl(struct xclmgmt_dev *lro, void *buf)
 	hwicap = (struct xcl_pr_region *)buf;
 	ret = xocl_clock_get_data(lro, CLOCK_FREQ_0);
 	if (ret == -ENODEV) {
-		hwicap->freq_0 = xocl_xgq_clock_get_data(lro, CLOCK_FREQ_0);
```

Comment:
No need to make this change, because it creates confusion with other code above. By design, this api can return any data, including clock counter numbers. Even though, we now only return clock counters, we don't have to tie the name with clock counter. So please un-change this changes.